hack/gen-versions/calico.go.tpl

```diff
@@ -149,6 +156,7 @@ var (
 		ComponentCalicoCNI,
 		ComponentCalicoCNIFIPS,
 		ComponentCalicoCNIWindows,
+        ComponentCalicoCSRInitContainer,
```

Comment:
Can you check the indentation on this? I've seen github show indentation oddly sometimes so I'm not sure if it is github or the changes.